getting-started/about-telerik-reporting.md

```diff
@@ -47,19 +47,17 @@ The report definition is the model that contains the declarations of all the rep
 
 To create a report, you need to generate a report definition file by using the supported [Telerik Report Designers]({%slug telerikreporting/designing-reports/report-designer-tools/overview%}).
 
-* If you design a report in the Report Designer for Visual Studio, the result is a class inheriting `Telerik.Reporting.Report`.
+* If you design a report in the Report Designer for Visual Studio, the result is a class inheriting `Telerik.Reporting.Report`. These are the so called __Type Report Definitions__. They may be referenced also as _CLR_ (Common Language Runtime), _CS_ (C Sharp) or _VB_ (Visual Basic) report definitions. These separate CLR types may be passed to the Reporting engine through the built-in [TypeReportSource](/api/telerik.reporting.typereportsource).
```

Comment:
Great, thank you Toshe. The link is returning 404, but I guess it will be ok on test and live docs

Reply:
yes, it will be OK, the API ref is built with a separate process